src/CommunityToolkit/Xamarin.CommunityToolkit/PlatformConfiguration/Multiplatform/StatusBar/StatusBar.shared.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.PlatformConfiguration.Multiplatform
+{
+	public static partial class StatusBar
+	{
+		/// <summary>
+		/// Sets the color of application's status bar
+		/// </summary>
+		public static readonly BindableProperty ColorProperty = BindableProperty.Create(
+			"Color", typeof(Color), typeof(StatusBar), Color.Default, propertyChanged: (_, _, newValue) => SetColor((Color)newValue));
+
+		/// <summary>
+		/// Sets style of application's status bar
+		/// </summary>
+		public static readonly BindableProperty StyleProperty = BindableProperty.Create(
+			"Style", typeof(StatusBarStyle), typeof(StatusBar), StatusBarStyle.Default, propertyChanged: (_, _, newValue) => SetStyle((StatusBarStyle)newValue));
+
+		/// <summary>
+		/// Needed for BindableProperty to work. Don't call this method.
+		/// </summary>
+		public static Color GetColor(BindableObject element) => throw new NotImplementedException();
+
+		/// <summary>
+		/// Needed for BindableProperty to work. Don't call this method.
+		/// </summary>
+		public static StatusBarStyle GetStyle(BindableObject element) => throw new NotImplementedException();
+
+		/// <summary>
+		/// Sets the color of application's status bar
+		/// </summary>
+		/// <param name="color">Color to set</param>
+		public static void SetColor(Color color) => PlatformSetColor(color);
+
+		/// <summary>
+		/// Sets style of application's status bar
+		/// </summary>
+		/// <param name="style">Style to set</param>
+		public static void SetStyle(StatusBarStyle style) => PlatformSetStyle(style);
+
+		static partial void PlatformSetColor(Color color);
+
+		static partial void PlatformSetStyle(StatusBarStyle style);
+	}
+
+	public enum StatusBarStyle
```

Comment:
since this is public, what do you think to move this enum (and others) to a separated file?

Reply:
I was using your PR as a guideline. Does you comment mean that `PopoverArrowDirection` also needs to be moved to a separate file?

https://github.com/xamarin/XamarinCommunityToolkit/pull/949/files#diff-9fdc661ce4c965ee9871e89532420732f4d0cf2c1d8be607a17d88ee35ce1aca

Reply:
Yes that file should be moved to a separeted file, we missed that.